twoliter/embedded/rpm2img

```diff
@@ -186,61 +186,73 @@ INVENTORY_DATA="$(jq --raw-output . <<<"${installed_rpms[@]}")"
 # Sort by package name and add 'Content' as top-level.
 INVENTORY_DATA="$(jq --slurp 'sort_by(.Name)' <<<"${INVENTORY_DATA}" | jq '{"Content": .}')"
 
-# Get the core kit version and vendor from external kit metadata.
+# Iterate through all kits used to build this variant.
 EXTERNAL_KIT_METADATA_PATH="${EXTERNAL_KITS_PATH}/external-kit-metadata.json"
-CORE_KIT_VERSION=$(jq --raw-output '.kit[]|select(.name == "bottlerocket-core-kit")|.version' "${EXTERNAL_KIT_METADATA_PATH}")
-CORE_KIT_VENDOR=$(jq --raw-output '.kit[]|select(.name == "bottlerocket-core-kit")|.vendor' "${EXTERNAL_KIT_METADATA_PATH}")
-# Set the path inside the build container to the core kit RPMs and repo.
-CORE_KIT_PATH="${EXTERNAL_KITS_PATH}/${CORE_KIT_VENDOR}/bottlerocket-core-kit/${ARCH}"
-
-if [[ -n "${CORE_KIT_VERSION}" && -n "${CORE_KIT_VENDOR}" ]]; then
-  # Query the bottlerocket-core-kit repo for a single package's Release in order
-  # to extract the commit that built the core kit.
-  CORE_KIT_GIT_SHA="$(dnf --repofrompath \
-    core-kit,file://"${CORE_KIT_PATH}" \
-    --repo=core-kit repoquery \
+mapfile -t kits < <(jq -r ".kit[].name" "${EXTERNAL_KIT_METADATA_PATH}")
+for kit in "${kits[@]}"; do
+  KIT_VERSION=$(jq --arg kit "${kit}" --raw-output '.kit[]|select(.name == $kit)|.version' "${EXTERNAL_KIT_METADATA_PATH}")
+  KIT_VENDOR=$(jq --arg kit "${kit}" --raw-output '.kit[]|select(.name == $kit)|.vendor' "${EXTERNAL_KIT_METADATA_PATH}")
+  if [[ -z "${KIT_VERSION}" ]]; then
+      echo "Failed to extract version of kit '${kit}'" >&2
+      exit 1
+  fi
+  if [[ -z "${KIT_VENDOR}" ]]; then
+      echo "Failed to extract vendor of kit '${kit}'" >&2
+      exit 1
+  fi
+
+  # Set the path inside the build container to the kit RPMs and repo.
+  KIT_PATH="${EXTERNAL_KITS_PATH}/${KIT_VENDOR}/${kit}/${ARCH}"
+
+  # Query the kit repo for a single package's Release in order
+  # to extract the commit that built the kit.
+  KIT_GIT_SHA="$(dnf --repofrompath \
```

Comment:
Offline discussion helped us realize this isn't used anymore.

Reply:
https://github.com/bottlerocket-os/twoliter/commit/44a2623a6419dc7b07e346175a425bcad2f88aa6 switched to pulling package info directly from the RPM itself, so we can remove this entirely :D